docs/deploying/nixos.md

```diff
@@ -39,6 +39,11 @@ The `flake.nix` and `default.nix` do not currently provide a NixOS module (contr
 welcome!), so [`services.matrix-conduit`][module] from Nixpkgs can be used to configure
 conduwuit.
 
+Beware! The module defaults to SQLite as a database backend. Conduwuit dropped SQLite support
```

Comment:
Can this be put in a dedicated header to be more visible?

```suggestion
### Conduit NixOS Config Module and SQLite

Beware! The module defaults to SQLite as a database backend. Conduwuit dropped SQLite support
```

Reply:
hm. then with this lemme also rephrase my sentence a bit

Reply:
should i squash it all down into one commit